### PR TITLE
LPS-199345 Select the first option after pressing tab if there isn't any value already selected

### PR DESCRIPTION
**Pull request by rebsilva**
Related Issue: https://liferay.atlassian.net/browse/LPS-203863

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Radio/Radio.es.js [hunk 1]
```diff
@@ -64,6 +68,14 @@ const Radio = ({
...
 						onChange({target: {value}});
 					}}
+					onKeyUp={(event) => {
+						const value = options[0].value;
+
+						if (!currentValue && event.keyCode === KEYCODES.TAB) {
+							setCurrentValue(value);
+							onChange({target: {value}});
+						}
+					}}
```
Comment (cgmonte) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Radio/Radio.es.js:78]: if `(!currentValue && event.keyCode === KEYCODES.TAB)` evaluates to false, then the value assignment is useless. I'd just move to the if body:

```
onKeyUp={(event) => {
	if (!currentValue && event.keyCode === KEYCODES.TAB) {
		const value = options[0].value;

		setCurrentValue(value);
		onChange({target: {value}});
	}
}}
```

**Review by cgmonte — COMMENTED** [review]
Hi @rebsilva, I am happy to get back at reviewing PRs :blush: I just made a small observation about your change. Let me know if it makes sense.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Radio/Radio.es.js (hunk 1 above)
Comment (rebsilva) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Radio/Radio.es.js:78]: hey @cgmonte, is great to have you back! :blush: what you said make sense, i will change and send again, thanks!

**Comment by rebsilva** [comment]
ci:test:sf

**Review by cgmonte — APPROVED** [review]
LGTM!

**Comment by rebsilva** [comment]
ci:test:relevant

**Comment by rebsilva** [comment]
ci:test:forms

**Comment by rebsilva** [comment]
ci:test:data-engine

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**